python.d/web_log.chart.py

```diff
@@ -328,7 +340,8 @@ def __init__(self, service):
                      'resp_time_upstream_avg': 0, 'unique_cur_ipv4': 0, 'unique_cur_ipv6': 0, '2xx': 0,
                      '5xx': 0, '3xx': 0, '4xx': 0, '1xx': 0, '0xx': 0, 'unmatched': 0, 'req_ipv4': 0,
                      'req_ipv6': 0, 'unique_tot_ipv4': 0, 'unique_tot_ipv6': 0, 'successful_requests': 0,
-                     'redirects': 0, 'bad_requests': 0, 'server_errors': 0, 'other_requests': 0, 'GET': 0}
+                     'redirects': 0, 'bad_requests': 0, 'server_errors': 0, 'other_requests': 0, 'GET': 0,
+                     'port_80': 0, 'port_443': 0}
```

Comment:
Why do we need this from start? It is added an empy chart (`'port_80': 0, 'port_443': 0` always 0) to all plugin users.
I suggest to not add zeroed `port_` keys here.

Reply:
OK, I remove this